docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc

```diff
@@ -17,7 +17,7 @@ You must upgrade the Compute services from {rhos_prev_long} {rhos_prev_ver} to {
 DEFAULT_CELL_NAME="cell1"
 RENAMED_CELLS="$DEFAULT_CELL_NAME"
 
-NAMESPACE="openstack"
+oc project "rhoso"
```

Comment:
You should reflect that on the test code as well, there is some logic build around NAMESPACE env var, so those must match the docs

Reply:
Yes but is the `oc project "rhoso"` scoped for OSPDO docs only? To me it looks like we're changing the default on non-OSPDO too here.